internal/cli/organizations.go

```diff
@@ -461,33 +464,140 @@ auth0 orgs members ls <id>`,
 				inputs.ID = args[0]
 			}
 
-			list, err := getWithPagination(
-				cmd.Context(),
-				inputs.Number,
-				func(opts ...management.RequestOption) (result []interface{}, hasNext bool, apiErr error) {
-					members, apiErr := cli.api.Organization.Members(url.PathEscape(inputs.ID), opts...)
-					if apiErr != nil {
-						return nil, false, apiErr
-					}
-					var output []interface{}
-					for _, member := range members.Members {
-						output = append(output, member)
-					}
-					return output, members.HasNext(), nil
-				})
+			members, err := cli.getOrgMembers(cmd.Context(), inputs.ID, inputs.Number)
+			if err != nil {
+				return err
+			}
+			sortMembers(members)
+			cli.renderer.MembersList(members)
+			return nil
+		},
+	}
+	return cmd
+}
+
+func rolesOrganizationCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "roles",
+		Short: "Manage roles of an organization",
+		Long:  "Manage roles of an organization.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listRolesOrganizationCmd(cli))
+	cmd.AddCommand(membersRolesOrganizationCmd(cli))
 
+	return cmd
+}
+
+func listRolesOrganizationCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID     string
+		Number int
+	}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "List roles of an organization",
+		Long:    "List roles assigned to members of an organization.",
+		Example: `auth0 orgs roles list
+auth0 orgs roles ls <id>`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := organizationID.Pick(cmd, &inputs.ID, cli.organizationPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			members, err := cli.getOrgMembers(cmd.Context(), inputs.ID, inputs.Number)
 			if err != nil {
-				return fmt.Errorf("Unable to list members of an organization with Id '%s': %w", inputs.ID, err)
+				return err
 			}
-			var typedList []management.OrganizationMember
-			for _, item := range list {
-				typedList = append(typedList, item.(management.OrganizationMember))
+
+			roleMap := make(map[string]management.OrganizationMemberRole)
+			for _, member := range members {
+				userID := member.GetUserID()
+				roleList, err := cli.api.Organization.MemberRoles(inputs.ID, userID)
```

Comment:
All this logic (that does network calls inside a given command) should be inside a single spinner block.